crates/goose/src/providers/bedrock.rs

```diff
@@ -62,16 +64,39 @@ impl BedrockProvider {
         set_aws_env_vars(config.all_values());
         set_aws_env_vars(config.all_secrets());
 
-        let sdk_config = aws_config::load_from_env().await;
+        // Use load_defaults() which supports AWS SSO, profiles, and environment variables
+        let mut loader = aws_config::defaults(aws_config::BehaviorVersion::latest());
+
+        // Check for AWS_PROFILE configuration
```

Comment:
nit: would it be ok to trim some of the one line comments, I know goose/others like to add them all the time, but they can add noise/mistakes in future as code drifts (ie only really needed ones - like the one on like 67 which talks about SSO is great).